python/cudf/cudf/_lib/cpp/copying.pxd

```diff
@@ -18,6 +18,8 @@ from cudf._lib.cpp.table.table cimport table
 from cudf._lib.cpp.table.table_view cimport table_view
 from cudf._lib.cpp.types cimport size_type
 
+ctypedef const scalar constscalar
```

Comment:
Do you really need to save typing a space?

Reply:
This is hopefully a temporary workaround. For some reason writing `cdef vector[reference_wrapper[const scalar]]` won't compile in cython, it raises some kind of syntax error. There's not much info about the problem online that I can find so far, but what I did find [suggest nuances in parsing template parameters](https://github.com/cython/cython/issues/534) might be the problem. Admittedly, I'm not really sure yet.